superset/connectors/base/models.py

```diff
@@ -110,6 +110,8 @@ def name(self) -> str:
     params = Column(String(1000))
     perm = Column(String(1000))
     schema_perm = Column(String(1000))
+    is_managed_externally = Column(Boolean, default=False)
+    external_url = Column(Text, nullable=True)
```

Comment:
couldnt we just compute the `is_managed_externally` based upon the if `external_url` ? instead of having a seperate column?

Reply:
The problem is that `external_url` should really be optional — depending on the source a resource might be managed externally, without having an external URL associated with it. So it might be possible to have `is_managed_externally=true` while at the same time `external_url=null`.